packages/components/Checkbox/src/Checkbox.settings.ts

```diff
@@ -3,14 +3,14 @@ import { checkboxName, ICheckboxType } from './Checkbox.types';
 
 export const checkboxSelectActionLabel = 'Toggle the Checkbox';
 
+/* Default values for Win32/Windows */
```

Comment:
If these are the windows/win32 default, would it be more proper to name this "settings.windows.ts" and have "settings.win32.ts" import from there?

Reply:
I understand this sentiment, however, I am trying to follow patterns done everywhere else in the repo. All the base files in this repo (whether it be each controls' .tsx file, .settings file, or .types file) are all Win32 specific. We haven't added .win32 specific files for any of these controls (to my knowledge).

Maybe, however, because we're introducing other platform-specific files in this folder, it would make sense to add .win32 specific files as well.

I think this should be a discussion we hold at an in-person meeting, as it will pave the repo structure moving forward as more platform-specific files come into play

Reply:
@Saadnajmi  Some extra info: I didn't actually change anything here. All I did was revert a couple of the things Tushar did in the last PR. The only thing I added was this comment to clarify this is Win32.

Reply:
@Saadnajmi I'm writing up a post to discuss the folder structure and what not, I just want to make sure we get this change in before tomorrow (the fork is on sunday and GDPR is in production and uses our Checkbox).

We need to have the Checkbox focus fix in asap for GDPR.

Reply:
And I want to make sure the broken version of Checkbox doesn't get pulled into Devmain for the fork

Reply:
I see. How about we check this in, with a followup conversation on naming so we don't block fixing this downstream?
If this is truly a revert, then I suppose the risk is minimal.

Reply:
Sounds good, I will make a task to discuss it! Thanks